src/System.Runtime/tests/System.Runtime.Tests.targets

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <Target Name="CopyFilesToNonTPADir" BeforeTargets="RunTestsForProject" >
```

Comment:
On further looking, I don't see a clean way to do this nicely without the custom target since to include it as supplemental test data you need to know its path, and to do that you need to binplace it to the root.  Adding extra outputpath to Test Assembly didn't help..    You could put the custom target into the test project though.